docs/src/main/sphinx/release/release-478.md

```diff
@@ -0,0 +1,100 @@
+# Release 478 (dd Oct 2025)
+
+## General
+
+* {{breaking}} Move startup logs to `io.trino.bootstrap` package. ({issue}`26721`)
+* Run Trino on JDK 25.0.0 (build 36). ({issue}`26693`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Fix failure when reading `NULL` map on `json` type. ({issue}`26700`)
```

Comment:
Json doesn’t have a concept of maps, so what what’s a null map?

Reply:
cc: @wendigo @chenjian2664

Reply:
it fixed the issue with reading map columns where the value type is json and the value is null
How about "Fix reading map type when value type is json and value is `null`"

Reply:
@chenjian2664 Please rephrase to clarify whether it was a failure or a correctness issue. It should start with either "Fix failure when ..." or "Fix incorrect results when …"

Reply:
Thanks.
It is a failure, so should be "Fix failure when reading map type with value type is json and value is `null`"